### PR TITLE
added extract model components

### PR DESCRIPTION
**Pull request by rosedoss**
Updated the 3.bate_vae_training.ipynb file to include the code to extract weights of the trained model as well as removed the code to create heatmap, which will be added to separate file.

Added an import call to vae_utils.py.

### CODE REVIEW

File: 2.train-VAE/scripts/nbconverted/3.beta_vae_training_extract_model_components.py [hunk 1]
```diff
@@ -196,116 +205,56 @@
...
 latent_df
 
 
 # In[21]:
 
 
-age_category = metadata.pop("age_category")
-sex = metadata.pop("sex")
-train_test = metadata.pop("train_or_test")
+class BaseModel():
```
Comment (gwaybio) [inline 2.train-VAE/scripts/nbconverted/3.beta_vae_training_extract_model_components.py]: I think you can delete this, right? I don't see this being used.

**Review by gwaybio — APPROVED** [review]
Looks great! 🤩  One minor comment.

Before you merge, please also add the contents of the `results/` directory. (weight matrix, latent data, training history). These files belong in this repo!

**Comment by rosedoss** [comment]
IMPORTANT: The commit named 'updated csv files from model' also contains changes to the 3.beta_vae_training_extract_model_components.ipynb file

**Review by gwaybio — APPROVED** [review]
What is `weight_matrix_gsea.csv`?

It's always good to make sure all the files you add have _provenance_, which means anyone can track their origin clearly and reproducibly.

Also, it looks like you removed the random seed? We should probably keep that in :D

**Comment by rosedoss** [comment]
@gwaybio 
the weight_matrix_gsea.csv is formatted to be used for the GSEA where the gene names are added back in under the '0' column. Should I add more detail or not include this file?

Also, there were two code lines that had the same random seed and I thought this was repetitive. I can add it back in though!

**Pull request merged**